### PR TITLE
build: making the build script distribution agnostic

### PR DESCRIPTION
**Pull request by ldeweysuse**
Signed-off-by: Larry Dewey <ldewey@suse.com>

### CODE REVIEW

**Comment by lilienbm** [comment]
test

**Comment by ldeweysuse** [comment]
@lilienbm let me know if there is any way that I can be of help.

**Comment by lilienbm** [comment]
Can you change the last line to say "# Rebuild SEV Tool binary" instead of Rename. That was my typo.

I think there's a bug in your changes. I'm testing this on Ubuntu, and I'm echoing out distbase and I'm getting nothing. So none of the IF cases are getting caught

**Comment by ldeweysuse** [comment]
I made the assumption that it would be okay to have this as a bash script, but if we want it as a Unix Shell script, I have a different version which we may go with instead.

**Comment by lilienbm** [comment]
2 more things :)
1. can we add an else case to the libelf thing:
   else
	   debug "Distribution not recognized. Not installing libelf dependency"
   fi
2. at line 23, the case of the variable was changed to capital letters, so the last line in the file needs to be changed too. It currently doesn't do the directory switch back

**Comment by ldeweysuse** [comment]
Good point about the libelf else statement. It was added, but I used echo in place of debug, as debug may or may not be turned on when the user is running the script. I also changed the variable usage on the last line to match it's declaration, so it should now return to the original directory.

As a tip, if you would like to add comments directly to the commit to suggest changes or address questions, you can do the following:

1. Click on the commit SHA on the right hand side (ex. 683e53b).
2. Once the page loads, while reviewing the commit, hover your mouse on the line you want to comment.
3. Click the blue '+' icon which appears.
4. Add your comment.

While your comment was very clear and easy to understand, this can provide context to someone which may be confused.

File: build.sh [hunk 1]
```diff
@@ -90,4 +93,5 @@ make clean
 make -j64
 cd ../
 
-cd $old_dir
+# Return to original directory
+cd ${OLD_DIR}
```
Comment (lilienbm) [inline build.sh]: ahhh

**Pull request merged**